packages/react/.env.example

```diff
@@ -8,7 +8,7 @@ SEARCH_REDIRECT_ENDPOINT=<endpoint for ibm.com search, e.g. https://www.ibm.com/
 
 # Search Typeahead
 SEARCH_TYPEAHEAD_VERSION=<api version for ibm.com search, e.g. v1>
-SEARCH_TYPEAHEAD_HOST=<host for ibm.com search, e.g. https://www-api.ibm.com>
+SEARCH_TYPEAHEAD_API=<host for ibm.com search, e.g. https://www-api.ibm.com>
```

Comment:
@kennylam we probably need to note this API change in our release notes (pinned issue)

Reply:
Done!